CHANGELOG.md

```diff
@@ -82,6 +82,7 @@ It features a lot of new rules from different categories.
 - Forbids to use and declare `float` keys in arrays and dictionaries
 - Forbids to use `a[len(a) - 1]` because it is just `a[-1]`
 - Forbids too long call chains like `foo(a)(b)(c)(d)`
+- Enforces stricter slice operation usage
```

Comment:
This should go to `0.15.0` section

Reply:
Could you be more specific? I don't see a section ```0.15.0``` in ```CHANGELOG.md```. It only goes up to ```0.13.3```

Reply:
That's what you have to create 🙂

Reply:
Okay! Could you explain reasoning why this is going into a new section rather than 0.13.0?

Reply:
Because `0.13` is already released, and we are working on `0.15`